content/04.results.md

```diff
@@ -1 +1,195 @@
+Figure @fig:fig2 shows an overview of the process and example input data for this analysis.
+This analysis relies upon accurate gender prediction of both authors and speakers.
+To predict the gender of the speaker or author, we used the package genderizeR [@https://github.com/kalimu/genderizeR], an R package wrapper to access the genderize.io API [@https://genderize.io/] to get binary gender predictions for each identified first name.
+We, unfortunately, cannot identify non-binary gender expression with the tools we used.
+To identify our binary gender prediction error rate, we created a benchmark data set of thirty randomly selected news articles; ten from each of the following years: 2005, 2010, 2015.
+In these articles, we hand-annotated the quoted speakers and their assumed gender.
+We then compared our hand-annotated gender prediction against the gender prediction after the entire scraping, coreNLP quote annotator, and genderizeR pipeline.
+Using our hand-annotated prediction as ground truth, we predicted a total of 55 female and 155 male observed names within the benchmark articles.
+We found that the gender prediction had an accuracy of 0.914, with a Kappa statistic of 0.767 (Supplemental Figure 1 a).
+Now that we have established reasonable accuracy for our gender prediction, we determined the rate of gendered quotes within articles.
+We first looked at the number of quotes identified within each news-related article (Figure @fig:fig2 b), totaling 120,309 quotes with a gender prediction for the speaker.
+As previously observed in article frequencies, we see quote frequencies vary by article type.
+The only article types in which there is a large discrepancy between the number of quotes and articles are "news-and-views" and "career-column.”
+This discrepancy is expected in "news-and-views" since this article type typically is an overview of a specific scientific topic and rarely quotes authors.
+Similarly, "career-column" is a perspective piece from a scientist who also rarely quotes other persons.
+
+We then compared the number of male quotes to the number of male first and last authors published in Nature (Figure @fig:fig2 c).
+We analyzed a total of 10466 first authors and 10494 last authors with a gender prediction.
+As denoted by the red line, we find that both authorship and quotes are far from gender parity.
+Additionally, we find a difference in author genders between first and last authors, with the last authors being more male-dominated.
+Since Nature authorship may not represent scientific publishing as a whole, we also compared against a random selection of authors from English language journals published by Springer (Supplemental Figure 2 a).
+We observed a more significant gender gap between first and last authors in our selection of Springer articles; however, both first and last authors are much closer to parity than Nature authors.
```

Comment:
```suggestion
We observed a larger gender gap between first and last authors in our selection of Springer articles; however, both first and last authors are much closer to parity than Nature authors.
```

Reply:
Nearly every journal now uses "significant" only for clauses that include a statistical test - I don't think (as written) we've done one for this difference.

Reply:
yes, this is true.